src/track/timesignature.h

```diff
@@ -8,39 +8,41 @@ namespace mixxx {
 /// Musical Time signature
 /// Right now is to be used for bar detection only, so only the beats per bar is useful
 class TimeSignature final {
-  private:
-    int m_beatsPerBar;
-    int m_noteValue;
-
   public:
-    TimeSignature(int beatsPerBar, int noteValue)
-            : m_beatsPerBar(beatsPerBar),
-              m_noteValue(noteValue) {
-    }
+    // The default constructor will set time signature to 4/4
+    // since it is defined as the default in beats.proto
```

Comment:
```suggestion
    // because it is defined as the default in beats.proto
```
"Since" means one event occurred before another in time.

Reply:
Now "I am going to bed since I have to get up early." ;)